app/models/archived_patient.rb

```diff
@@ -94,15 +94,15 @@ def self.convert_patient(patient)
 
     )
 
-    archived_patient.build_fulfillment(dup_with_context(patient.fulfillment)).save
+    archived_patient.build_fulfillment(patient.fulfillment.attributes.except('id')).save
```

Comment:
I wrote a custom `dup_with_context` thing in the original draft of this (and then didn't port it over into the archived patient pr 🤦 ) but this basically does the same thing, so might as well not delegate it to its own method.

Reply:
wow I should have caught that 😅

Reply:
no harm no foul, usually that's the job of tests!